pkg/storaged/btrfs/subvolume.jsx

```diff
@@ -242,7 +245,70 @@ function subvolume_delete(volume, subvol, mount_point_in_parent, card) {
     });
 }
 
-export function make_btrfs_subvolume_page(parent, volume, subvol) {
+function dirname(path) {
+    const i = path.lastIndexOf("/");
+    if (i < 0)
+        return null;
+    else
+        return path.substr(0, i);
+}
+
+export function make_btrfs_subvolume_pages(parent, volume) {
+    let subvols = client.uuids_btrfs_subvols[volume.data.uuid];
+    if (!subvols) {
+        const block = client.blocks[volume.path];
+        /*
+         * Try to show subvolumes based on fstab entries, this is a bit tricky
+         * as mounts where subvolid cannot be shown userfriendly.
```

Comment:
This line confuses me everything I read it. `this is a bit tricky as mounts where subvolid cannot be shown userfriendly`?? Don't you mean, in fstab we might not have the subvolid's (they can be there but don't have too) so we have to make something up as described below.

Reply:
Yeah, same here. I don't think I wrote this... :-)  I'll try to make this better.